guides/inotify-watch-limits.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Inotify Watch Limit Increase
+description: Learn how to increase the inotify watcher limit.
+---
+
+When running a webpack project, you may encounter an error similar to the
+following:
+
+```text
+Watchpack Error (watcher): Error: ENOSPC: System limit for number of file
+watchers reached, watch '/some/path'
+```
+
+This results from a low number of [inotify
```

Comment:
Suggest adding more info about what the inotify mechanism is for, what it does, and the impact of changing the watch limit.

```suggestion
The [inotify facility](https://en.wikipedia.org/wiki/Inotify) allows programs to monitor files and directories for changes, so that they will receive an event immediately whenever a user or program modifies the file or directory. Because the inotify mechanism requires some kernel resources (memory and processor) per watched file, there is a limit on the number of registered file watchers. For large projects with many files and folders, programs may hit this limit, resulting in the above error message.
```